bigbone/src/main/kotlin/social/bigbone/api/method/admin/AdminMeasuresMethods.kt

```diff
@@ -0,0 +1,226 @@
+package social.bigbone.api.method.admin
+
+import social.bigbone.MastodonClient
+import social.bigbone.MastodonRequest
+import social.bigbone.Parameters
+import social.bigbone.api.entity.admin.AdminMeasure
+import social.bigbone.api.entity.admin.AdminMeasure.Key
+import java.time.Instant
+
+/**
+ * Obtain quantitative metrics about the server.
+ * @see <a href="https://docs.joinmastodon.org/methods/admin/measures/">Mastodon admin/measures API methods</a>
+ */
+class AdminMeasuresMethods(private val client: MastodonClient) {
```

Comment:
To remain consistent, we should rename this class to `AdminMeasureMethods`.

Reply:
But it’s the `/measures` API, shouldn’t it be `MeasuresMethods` then? 🤔

Reply:
I know, we had long discussions about this and then agreed on using the singular version of the word.

Reply:
Is this about singular vs. plural, or about the "Admin" prefix vs. no prefix?

Reply:
About the singular VS plural. I would rather name it `AdminMeasuresMethods` as it exposes the `/admin/measures` methods.

Reply:
FWIW, the discussion André mentioned happened in #94.

Reply:
Personally, I don't like class names like `DomainBlocksMethods`, `ConversationsMethods`, etc that much, as they do not go off the tongue as easily as their colleagues `DomainBlockMethods` or `ConversationMethods`. Latter seems more natural to me. But yes... We can vote, an it seems you guys want to change it. So feel free to do it. 👍

Reply:
> not go off the tongue as easily

I never need to actually speak them, so I don’t mind that. 😁 

I’d prefer naming the methods classes like the endpoints are named, but we’re close enough so I also wouldn’t mind it too much if we kept it as-is.